tools/dockerfile/grpc_clang_format/Dockerfile

```diff
@@ -12,12 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:15.10
+FROM debian:jessie
 
 RUN apt-get update && apt-get -y install wget
-RUN echo deb http://llvm.org/apt/wily/ llvm-toolchain-wily-3.8 main >> /etc/apt/sources.list
-RUN echo deb-src http://llvm.org/apt/wily/ llvm-toolchain-wily-3.8 main >> /etc/apt/sources.list
-RUN wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key| apt-key add -
+RUN echo "deb http://llvm.org/apt/jessie/ llvm-toolchain-jessie-3.8 main" >> /etc/apt/sources.list
```

Comment:
remark: this could potentially change the formatting enforced by clang_format, but if sanity is passing, it looks like the formatting being enforced is the same, so it seems safe to do.

Reply:
Yeah, I think it's safe since the clang versions should be the same, and if not, the clang formatting test should've said something.